in_toto/keylib.go

```diff
@@ -384,9 +384,9 @@ func getDefaultKeyScheme(key interface{}) (scheme string, keyIDHashAlgorithms []
 
 func (k *Key) loadKey(key interface{}, pemData *pem.Block, scheme string, keyIDHashAlgorithms []string) error {
 
-	switch key.(type) {
+	switch K := key.(type) {
```

Comment:
this does not need to be capital "K". No need to make it exported. Can we use another variable name if it conflicts with the function.